### DISCUSSION TITLE
啥时候新功能提案能全做完呢?

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1887] [讨论]
如题 (笑

**Comment by so1ve** [comment]
别急着催更

**Comment by WForst-Breeze** [comment]
开发者会优先处理票数较高、功能收益较高的内容，无特殊情况的话这些功能都会制作。不必着急，耐心等待即可

**Comment by Chiloven945** [comment]
~全部做完的时候~
开发是有周期的，不可能一下子全部就做完，而且龙猫需要把精力放在重要的功能上，不可能啥都做

**Comment by LTCatt** [comment]
等到鸡吃完了米，狗舔完了面，火烧断了锁，G 胖数 3，丝之歌发售
